builtin-contract/crosschain/contracts/crosschain.sol

```diff
@@ -36,7 +36,13 @@ contract CrossChain is Context, EIP712 {
     event CrossFromCKB(address to, address token, uint256 amount);
```

Comment:
Add the origin ckb tx hash in the event.

Reply:
done

Reply:
> Add the origin ckb tx hash in the event.

done